src/common/locations.ts

```diff
@@ -186,3 +196,48 @@ export function getColleges(fips: string): CollegeData[] {
 export function getStateCode(stateName: string) {
   return REVERSED_STATES[stateName];
 }
+
+const ALL_LOCATIONS = getLocationNames();
```

Comment:
FWIW (probably not worth worrying about in this PR)-
1. `getLocationNames` should probably be `getLocations` or similar?
2. I think we should do a pass at some point to try to unify all of our code to work with `State`, `County`, and `Location` data types and try to make sure all related logic is in this file and properly shared across parts of the site.  Right now it's pretty fragmented / ad-hoc.

Reply:
agreed, we should try to unify that at some point